update: kapacitor remove python2 examples (#6126)
influxdata/docs-v2: content/kapacitor/v1/guides/anomaly_detection.md

kapacitor define print_temps -tick print_temps.tick
```

Ensure that the task is enabled:

```bash
kapacitor enable print_temps
```

> And then...

Reviewer: ```suggestion
And then list the tasks:
```

```sh
# Start the recording in the background
kapacitor record stream -task print_temps -duration 24h -no-wait

> # Grab the ID and store it in a var

Reviewer: ```suggestion
# Copy the ID and store it in a variable
```

> # Copy the ID and store it in a variable
> kapacitor list recordings
> ID                                   Type    Status    Size      Date
> 7bd3ced5-5e95-4a67-a0e1-f00860b1af47 stream  running   0 B       04 May 16 11:34 MDT
> 

Reviewer 2: ```suggestion
# List recordings to find the ID
kapacitor list recordings
ID                                   Type    Status    Size      Date
7bd3ced5-5e95-4a67-a0e1-f00860b1af47 stream  running   0 B       04 May 16 11:34 MDT
# Copy the ID and store it in a variable
```
